src/components/Article.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Article({ title, date, text }) {
+  return (
+    <>
+      <h1>{title}</h1>
+      <span>{date}</span>
+      <p>{text}</p>
+    </>
+  );
+}
+
+PropTypes.objectOf(PropTypes.string);
```

Comment:
this line does nothing,
check here how to declare propTypes  https://reactjs.org/docs/typechecking-with-proptypes.html